### DISCUSSION TITLE
how to get code-server dmp file

### DISCUSSION THREAD

**Discussion by wanghehui1997** [coder/code-server#6950] [Q&A]
What command does code-server have to generate *.dmp files when the code-server process crashes, or in other words, how can I obtain crash information for code-server?

**Comment by code-asher** [comment]
code-server does not have anything special for this, but it is just a Node program so you should be able to use any flags you want by invoking Node directly, for example:

```
node --report-on-fatalerror /path/to/code-server/lib
```

But I think core dump files are controlled by the OS?
